ui/src/app/dashboard/grouped-summary/grouped-summary.component.ts

```diff
@@ -37,7 +37,10 @@ export class GroupedSummaryComponent {
 
   getCommonUrlParamsMap(entry: AggregationEntry) {
     let map = new Map<string, string[]>();
-    map.set('projectId', [this.activatedRoute.snapshot.queryParams['projectId']]);
+
+    const projectId = URLSearchParamsUtils.unpackURLSearchParams(this.activatedRoute.snapshot.queryParams['q']).extensions['projectId'];
```

Comment:
Can you wrap this into two lines its pretty long

Reply:
Fixed!